lib/i18n.js

```diff
@@ -126,18 +130,18 @@
         });
       },
 
-      _scheduleObservers: Ember.on('init', function() {
+      _scheduleObservers: function() {
```

Comment:
Not all apps have prototype extensions enabled. The official policy is that addons *shouldn't* rely on them.